packages/vaadin-virtual-list/test/wheel-scrolling.test.js

```diff
@@ -0,0 +1,195 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, nextRender } from '@vaadin/testing-helpers';
+import Sinon from 'sinon';
+import { Virtualizer } from '../virtualizer';
+
+function canScroll(el, deltaY) {
+  return (
+    (['auto', 'scroll'].indexOf(getComputedStyle(el).overflow) !== -1 &&
+      deltaY > 0 &&
+      el.scrollTop < el.scrollHeight - el.offsetHeight) ||
+    (deltaY < 0 && el.scrollTop > 0)
+  );
+}
+
+describe('scrolling mode', () => {
+  const IGNORE_WHEEL_TIMEOUT = 500;
+  let wrapper;
+  let virtualizer;
+  let scrollTarget;
+  let clock;
+  let child;
+  let grandchild;
+
+  beforeEach(() => {
+    wrapper = fixtureSync(`
+      <div style="height: 100px; overflow: auto;">
+        <div style="height: 200px;">
+          <div></div>
+        </div>
+      </div>
+    `);
+    scrollTarget = wrapper.firstElementChild;
+    const scrollContainer = scrollTarget.firstElementChild;
+
+    const wheelListener = (e) => {
+      if (!e.defaultPrevented && canScroll(e.currentTarget, e.deltaY)) {
+        e.currentTarget.scrollTop += e.deltaY;
+        e.preventDefault();
+      }
+    };
+
+    wrapper.addEventListener('wheel', wheelListener);
+
+    virtualizer = new Virtualizer({
+      createElements: (count) => Array.from(Array(count)).map(() => document.createElement('div')),
+      updateElement: (el, index) => {
+        el.index = index;
+        el.id = `item-${index}`;
+
+        if (!el.firstElementChild) {
+          el.innerHTML = `
+            <div class="child" style="height: 20px; overflow: auto;">
+              <div class="grandchild">
+                <div class="content" style="height: 40px;">
+                </div>
+              </div>
+            </div>
+          `;
+          el.querySelector('.child').addEventListener('wheel', wheelListener);
+          el.querySelector('.grandchild').addEventListener('wheel', wheelListener);
+        }
+
+        el.querySelector('.content').textContent = index;
+      },
+      scrollTarget,
+      scrollContainer
+    });
+
+    virtualizer.size = 100;
+
+    child = scrollContainer.firstElementChild.querySelector('.child');
+    grandchild = scrollContainer.firstElementChild.querySelector('.grandchild');
+
+    clock = Sinon.useFakeTimers();
+  });
+
+  afterEach(() => {
+    clock.restore();
+  });
+
+  function wheel({
```

Comment:
Created an issue to add this to `@vaadin/testing-helpers` https://github.com/vaadin/testing-helpers/issues/6